core/market/src/protocol/discovery.rs

```diff
@@ -147,8 +153,9 @@ impl Discovery {
 
     async fn on_bcast_offers(self, caller: String, msg: OffersBcast) -> Result<(), ()> {
         let num_ids_received = msg.offer_ids.len();
-        if !msg.offer_ids.is_empty() {
-            log::trace!("Received {} Offers from [{}].", num_ids_received, &caller);
```

Comment:
As far as I remember this log was only for non empty cases, because it was to much spam. But maybe it is not a problem as long it is on trace, so you can ignore this comment if you want.

Reply:
correct, but now it SHOULD not send empty bcasts again so we should see it when it does happen